packages/markdown/components/RdmeWrap/index.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
+const React = require('react');
+// const PropTypes = require('prop-types');
+
+require('./style.scss');
+
+const RdmeWrap = ({attributes, children}) => {
+  function test({target}, index) {
```

Comment:
Is this needed?

Reply:
Yeah, it's the tabbed code editor—just absolutely terribly named!